controllers/kafkacluster_controller.go

```diff
@@ -183,6 +192,10 @@ func (r *KafkaClusterReconciler) Reconcile(ctx context.Context, request ctrl.Req
 		}
 	}
 
+	if err := cruiseControlHealer.EnableSelfHealing(); err != nil {
```

Comment:
This should be done conditionally only if cruisecontrol.properties enablea the selgf healer.
Upstream in banzaicloud the healer is disabled by default and we should not enable in that case

Reply:
It is. The check is inside the function though. https://github.com/adobe/koperator/pull/50/files#diff-eb931b9f4daf9255b1ed81d4ccec566fa1dd318ce82f0359c2afefefdd696453R98. I should rename the function or leave a comment

Reply:
Got it!